language/class_spec.rb

```diff
@@ -7,6 +7,18 @@ module ClassSpecs
   Number = 12
 end
 
+describe "The class keyword" do
+  it "creates a new class with semicolon" do
+    class C; end
+    C.should be_an_instance_of(Class)
+  end
+
+  ruby_version_is "2.3" do
+    it { lambda { eval "class C end" }.should_not raise_error(SyntaxError) }
```

Comment:
`does not raises a SyntaxError when opening a class without a semicolon` sounds good